#hipnuc config
imu_serial: "/dev/ttyUSB0"
baud_rate: 115200